auraed/src/runtime/cells/cell_service.rs

```diff
@@ -98,44 +102,100 @@ impl CellService {
     async fn start(
         &self,
         request: ValidatedStartExecutableRequest,
-    ) -> Result<StartExecutableResponse> {
-        let ValidatedStartExecutableRequest { cell_name, executable } = request;
+    ) -> std::result::Result<Response<StartExecutableResponse>, Status> {
```

Comment:
maybe next iteration: this should probably be on ExecutableService as it should be possible to start an executable without a cell with auraed.

Reply:
Confirming my agreement from discord: I like the idea of an ExecutableService.

I'm not sure about an executable without a cell (that is, the top auraed, true pid 1, maybe shouldn't have that ability). I thought I read that systemd runs everything in a cgroup. I assume there is a reason? That is something I'm sure you and Nóva can answer.

Reply:
it does yes (https://opensource.com/article/20/10/cgroups) which allows "per service" resource allocation.  

i model the difference as the `ExecutableService` is the "just run this thing for me" service and the `CellService` is the "run this thing for me with this resource constraint" service.  perhaps we don't need to separate these and everything is run in a Cell but with optional constraints.  this just means we don't need to iterate on your work here :D

my conceptual framework is based on previous discussions around the API where we had "executable", "cell", and "pod" as different layers of "running a thing".  maybe the first is no longer necessary.